### PR TITLE
Vinisha, Kirity | MOBN-1475 | Clear status state and auth state on logout and wrap subheader message in Notification card

### PR DESCRIPTION
**Pull request by vinishad**
# JIRA TICKET NAME:
[MOBN-1475](https://msfprojects.atlassian.net/browse/MOBN-1475)

# Summary:

### CODE REVIEW

File: src/components/NotificationCard.tsx [hunk 1]
```diff
@@ -74,7 +74,7 @@ const NotificationCard: React.FC<Props> = ({
                     >
                         {headerMessage}
                     </Typography>
-                    <Typography variant="subtitle2" color="textSecondary">
+                    <Typography noWrap variant="subtitle2" color="textSecondary">
```
Comment (rakesh-1010) [inline src/components/NotificationCard.tsx:77]: @vinishad, can you revert this change. We have changed the folder structure, and this will cause merge conflicts. We will add this change as part of our card.

Comment (kirity-mahanty) [reply src/components/NotificationCard.tsx:77]: @rakesh-1010 This change is part of this card. You can merge this PR. After that you can rebase your branch with this to avoid the merge conflict.
